Content.Server/Buckle/Components/BuckleComponent.cs

```diff
@@ -230,8 +230,9 @@ public override bool TryBuckle(EntityUid user, EntityUid to)
 
             UpdateBuckleStatus();
 
-            var ev = new BuckleChangeEvent() { Buckling = true, Strap = BuckledTo.Owner };
+            var ev = new BuckleChangeEvent() { Buckling = true, Strap = BuckledTo.Owner, BuckledEntity = Owner };
             _entMan.EventBus.RaiseLocalEvent(Owner, ev, false);
+            _entMan.EventBus.RaiseLocalEvent(BuckledTo.Owner, ev, false);
```

Comment:
Using an event dual purpose like this is ugly. It should be an event for strap instead.